.github/workflows/release.yaml

```diff
@@ -138,3 +138,41 @@ jobs:
           args: release --clean --config .goreleaser/${{ env.GO_RELEASER_CONFIG }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get new chart version from the chart repo
+        id: get_chart_version
+        run: |
+          helm repo add kusionstack https://kusionstack.github.io/charts
+          helm repo update
+          version=$(helm search repo kusionstack/karpor --versions | head -n 2 | tail -n 1 | awk '{print $2}')
+          if [ -z "$version" ]; then
+            echo "Error: Unable to fetch chart version" >&2
+            exit 1
+          fi
+          echo "Current chart version is: $version"
+          major=$(echo "$version" | cut -d. -f1)
+          minor=$(echo "$version" | cut -d. -f2)
+          patch=$(echo "$version" | cut -d. -f3)
+          new_chart_version="${major}.${minor}.$((patch + 1))"
+          echo "New chart version is: $new_chart_version"
+          echo "::set-output name=new_chart_version::$new_chart_version"
+
+      - name: Bump version in the related HelmChart Chart.yaml
+        uses: fjogeleit/yaml-update-action@main
+        with:
+          repository: KusionStack/charts
+          valueFile: 'charts/karpor/Chart.yaml'
+          propertyPath: '{"version":"${{ steps.get-chart-version.outputs.new_chart_version }}", "appVersion":"${{ steps.get_version.outputs.VERSION }}"}'
```

Comment:
It should be modify to `steps.get_chart_version.outputs.new_chart_version`, otherwise the `version` cannot be obtained.

![image](https://github.com/user-attachments/assets/d287a285-430e-4fb7-83e0-25c43462d5ea)